### ISSUE TITLE
can not detect longitude

### ISSUE THREAD

**Issue by chewarum** [dmbfm/unity-ar-gps-location-issues#73]
![Screenshot_20200801-163546_UrbanGPS](https://user-images.githubusercontent.com/38130149/89097021-20a0ce80-d416-11ea-872a-acdefc487034.jpg)

I cant detect longitude its not my device issue because other map application works well...
is there any solutions???

and based on the time when I start the application, it is created only on the left side.
when I face north it spawns at west, when I face west it spawns east

**Comment by dmbfm** [comment]
> I cant detect longitude
It says your longitude is 50.699 and latitude 37.04, is this not correct?

>and based on the time when I start the application, it is created only on the left side.
>when I face north it spawns at west, when I face west it spawns east
Can you check what the "Focs Mode" is set on the "AR Camera"? I've seen some people having similar issues if the 
focus is set to "Auto".

**Comment by chewarum** [comment]
> > I cant detect longitude
> 
> It says your longitude is 50.699 and latitude 37.04, is this not correct?
> 
> > and based on the time when I start the application, it is created only on the left side.
> > when I face north it spawns at west, when I face west it spawns east
> 
> Can you check what the "Focs Mode" is set on the "AR Camera"? I've seen some people having similar issues if the
> focus is set to "Auto".

![image](https://user-images.githubusercontent.com/38130149/89447673-54228680-d791-11ea-9c5a-a10459c4b658.png)
do yo mean this?

**Comment by dmbfm** [comment]
Yes, disable "Auto Focus" to see if it makes any difference in the results.
